extensions/odoo_theme/layout_templates/legal.html

```diff
@@ -0,0 +1,150 @@
+                            {%- if language_code != 'en' %}
+                                <a class="btn btn-light text-primary" target="_blank" href="legal/terms/enterprise.html">
+                                    English
+                                </a>
+                            {%- endif %}
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="col-sm-4">
+                <div class="card h-100">
+                    <div class="card-body d-flex flex-column justify-content-between">
+                        <div class="d-flex align-items-center mb-3">
+                            <i class="i-article"></i>
+                            <h5 class="card-title mb-0">{{ _("Odoo Partnership Agreement") }}</h5>
+                        </div>
+                        <div>
+                            {%- set partnership_agreement_path = 'legal/terms/i18n/partnership_%s.html' % language_code %}
+                                <a class="btn btn-primary" target="_blank" href="{{ partnership_agreement_path }}" role="button">{{ _("Read") }}</a>
+                            {%- if language_code != 'en' %}
+                            <a class="btn btn-light text-primary" target="_blank" href="legal/terms/partnership.html">
+                                English
+                            </a>
+                            {%- endif %}
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="col-sm-4">
+                <div class="card h-100">
+                    <div class="card-body d-flex flex-column justify-content-between">
+                        <div class="d-flex align-items-center mb-3">
+                            <i class="i-article"></i>
+                            <h5 class="card-title mb-0">{{ _("Terms Of Sale") }}</h5>
+                        </div>
+                        <div>
+                            {%- set terms_of_sale_agreement_path = 'legal/terms/i18n/terms_of_sale_%s.html' % language_code %}
+                                <a class="btn btn-primary" target="_blank" href="{{ terms_of_sale_agreement_path }}" role="button">{{ _("Read") }}</a>
+                            {%- if language_code != 'en' %}
+                            <a class="btn btn-light text-primary" target="_blank" href="legal/terms/terms_of_sale.html">
+                                English
+                            </a>
+                            {%- endif %}
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <dl class="footnote brackets">
+            <dt class="label" id="id2">
+                <span class="brackets">
+                    <a class="fn-backref" href="#id1">1</a>
```

Comment:
Clicking on this breaks the [1] in the card:

![image](https://user-images.githubusercontent.com/15896813/205101301-72cd8c23-2933-4329-959c-0882691f1c6a.png)